firmware_io/simulink_output.csv

```diff
@@ -1 +1,24 @@
-Variable Name, Datatype, Minimum Value, Maximum Value, Name Description, Value Description
+Variable Name,Datatype,Unit,Minimum Value,Maximum Value,Name Description,Value Description,
+DI_BrakeLightEn,bool,,0,1,Command to brake light,0 - Light off,1 - Light On
+DI_DriverSpeaker,bool,,0,1,Command to ready to drive speaker,0 - Speaker off,1 - Light On
+DI_PWMstatusLightCycle,uint,%,0,100,PWM percentage of start button light,0 - 0% PWM On,100 - 100% PWM On
+DI_PWMstatusLightFreq,uint,%,0,1000,Freq of start button light,0 - Off,1000 - Max Freq
```

Comment:
what is the unit here? can't be % if max=1000

Reply:
Changed to Hz

Reply:
This signal is unit less, please refer to page 61 on [AMK Datasheet](https://mcmasteru365.sharepoint.com/:b:/r/sites/MACFormulaElectric686/Shared%20Documents/ENG%20-%20Software%20and%20Vehicle%20Controls/AMK_Documentation/[AMK_Motor_Datasheet_2020_44.pdf](https://mcmasteru365.sharepoint.com/:b:/r/sites/MACFormulaElectric686/Shared%20Documents/ENG%20-%20Software%20and%20Vehicle%20Controls/AMK_Documentation/AMK_Motor_Datasheet_2020_44.pdf?csf=1&web=1&e=J0kZQu)?csf=1&web=1&e=J0kZQu)